src/CONST.ts

```diff
@@ -1101,7 +1101,11 @@ const CONST = {
     CUSTOM_UNITS: {
         NAME_DISTANCE: 'Distance',
         DISTANCE_UNIT_MILES: 'mi',
+        DISTANCE_UNIT_MILES_FULL_SINGULAR: 'mile',
```

Comment:
Move these to the language files. `CONST` should only be used for things that are accessed by code and the language files should be used when it's something displayed to a user.